js/game.js

```diff
@@ -0,0 +1,69 @@
+/** @type {HTMLCanvasElement} */
+
+class Game{
+    constructor(ctx, width, height, player){
+        this.ctx = ctx
+        this.width = width
+        this.height = height
+        this.player = player
+        this.intervalId = null
+        this.frames = 0
+        this.enemies = []
+    }
+
+    start(){
+        this.intervalId = setInterval(this.upDate, 1000/60) 
+    }
+
+     upDate = () => {
+        this.frames++
+        this.clear()
+        this.player.newPosition()
+        this.player.draw()
+        this.updateEnemies()
+        this.checkGameOver()
+    }
+
+    stop(){
+        clearInterval(this.intervalId)
+    }
+
+    clear(){
+        this.ctx.clearRect(0, 0, this.width, this.height)
+    }
+
+    updateEnemies(){
+        for(let i = 0; i < this.enemies.length; i++){
+            this.enemies[i].y += 3
+            this.enemies[i].draw()
+        }
+
+        if(this.frames % 120 === 0){
+            let randomSize = Math.floor(Math.random() * 150 - 40) + 40
+            let randomX = Math.floor(Math.random() * this.width - randomSize) + randomSize
```

Comment:
Here, maybe you could predefine the `height` of the enemies since they are supposed to be like 'rows' or so, and randomize the `width` and the `x` values. Always taking in consideration that you need to leave space for the car to pass.